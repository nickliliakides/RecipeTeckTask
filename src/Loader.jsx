import React from 'react';
import './Loader.styles.scss';

const Loader = () => {
  return (
    <div className='loader'>
      <svg>
        <use href='img/icons.svg#icon-cw'></use>
      </svg>
    </div>
  );
};

export default Loader;
