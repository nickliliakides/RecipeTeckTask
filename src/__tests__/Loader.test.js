import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../Loader';

let wrapper;

describe('<Loader /> component', () => {
  beforeEach(() => {
    wrapper = shallow(<Loader />);
  });

  afterEach(() => {
    wrapper.unmount();
  });

  it('should matches its snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
