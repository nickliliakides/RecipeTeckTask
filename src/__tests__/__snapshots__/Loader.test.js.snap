// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> component should matches its snapshot 1`] = `
<div
  className="loader"
>
  <svg>
    <use
      href="img/icons.svg#icon-cw"
    />
  </svg>
</div>
`;
